CompetitionTemplate/src/test/java/competition/subsystems/elevator/commands/RiseCommandTest.java

```diff
@@ -0,0 +1,42 @@
+package competition.subsystems.elevator.commands;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import competition.subsystems.elevator.ElevatorSubsystem;
+import edu.wpi.first.wpilibj.MockXboxControllerAdapter;
+import xbot.common.controls.actuators.mock_adapters.MockCANTalon;
+import xbot.common.injection.BaseWPITest;
+
+public class RiseCommandTest extends BaseWPITest{
+	
+	RiseCommand command;
+	ElevatorSubsystem  elevator;
+	
+	@Override
+	public void setUp() {
+		// TODO Auto-generated method stub
```

Comment:
Anytime you see this ` // TODO Auto-generated method stub ` you should delete it.